hawkbit-rest-core/pom.xml

```diff
@@ -48,7 +48,6 @@
       <dependency>
          <groupId>javax.servlet</groupId>
          <artifactId>javax.servlet-api</artifactId>
-         <scope>provided</scope>
```

Comment:
I'd prefer a general use of `provided` scope of the servlet-api. Using `compile` scope will lead to problems when using different application-server. You can also build an hawkBit application as WAR layout and deployed in a application server your choice. This will lead to clashes as you mentioned. So I'd prefer `provided` also for other modules as well :smile: